reference/commands/consumer/install.rst

```diff
@@ -312,6 +322,28 @@ With the :command:`-o` parameters you can only define specific package options.
     You can use :ref:`profiles <profiles>` files to create predefined sets of **settings**,
     **options** and **environment variables**.
 
+conf
+----
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+With the :command:`-c` parameters you can define specific package configurations.
```

Comment:
Being pedantic: "config" defines not "package" configuration, but rather "tool" one. It's not per-package style (like settings or options `zlib:tools.cmake.cmaketoolchain:generator=Xcode`), is it?

Reply:
Actually, it is, it would totally make sense to force one generator to a specific package but not the others, wouldn't it? For example, you could use Ninja generator as default, but Visual Studio one for a certain package that you need to open the IDE for working on it.

Reply:
> Actually, it is, it would totally make sense to force one generator to a specific package but not the others, wouldn't it?

Yes. Another example is if some package can't be built with certain generator or in parallel.

But I'm not requesting this feature :). My suggestion was to replace word "package" with word "tool" because current text looks lika an incorrect copy-paste...

Reply:
Ah, yes, you are right! I misunderstood it :)

Please @franramirez688, a quick change to this would be nice.